tests/multipleBackend/objectPutCopyPart.js

```diff
@@ -152,6 +177,29 @@ describeSkipIfE2E('ObjectCopyPutPart API with multiple backends', () => {
         });
     });
 
+    it('should copy part to AWS based on mpu location', done => {
+        copyPutPart('mem', awsLocation, null, 'localhost', uploadId => {
+            assert.strictEqual(ds.length, 2);
+            const awsReq = Object.assign({ UploadId: uploadId }, awsParams);
+            s3.listParts(awsReq, (err, partList) => {
+                assertPartList(partList, uploadId);
+                s3.abortMultipartUpload(awsReq, err => {
```

Comment:
It can't be in the after step because the mpu upload id is one of the parameters